docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Ibexa Engage helps to engage audiences and collect valuable data by using interactive content.
+---
+
+# Ibexa Engage
+
+[[= product_name_engage =]] is a data collection tool. It gives you the ability to use the [Qualifio](https://qualifio.com/) tools to engage your audiences.
+You can use interactive content to build relationships and collect important data, for example, a list of recent orders, or personal information about customers.
+
+## Request access
+
+To use [[= product_name_engage =]], you must make arrangements with [[= product_name =]] to define the initial configuration, and then get and set up user account.
+
+1\. Go to the Back Office.
+
+2\. On the left panel, go to **Ibexa Engage**.
+
+3\. On the welcome page, provide the required information:
+
+- First and last name
+- Email address
+
+4\. Accept **Terms and Conditions** checkbox, and then click **Send request**.
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name_base =]] team creates and provides user account. An invitation link is sent during the setup process.
+Once the registration and configuration process is completed, the welcome page displays information about campaign tools available with [[= product_name_engage =]] and link to [Qualifio documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Campaigns) where you can get information about creating and managing campaigns.
+
+![Welcome page - active account](img/welcome_page_activated.png "Welcome page - active account")
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
+
+![Qualifio Manager](img/qualifio_manager.png "Qualifio Manager - campaign list")
+
+## Campaign
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
```

Comment:
Why do you say "you" in one sentence and "users" in the other? Is it that "you" is the campaign administrator and "users" are regular editors?

If there is no distinction betweeh "you" and "users", i'd say "You can also configure backgrounds..."